AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -243,7 +243,9 @@ define([
          * @param {Object} record
          */
         selectRecord: function (record) {
-            uiRegistry.get('name =' + this.mediaGallery).selected(record);
+            uiRegistry.get('name =' + this.mediaGallery)
+                .selected(record)
+                .toggleAddSelectedButton();
```

Comment:
I think it migth be better to use the `select` method that is responsible for handling that action
```suggestion
            uiRegistry.get('name =' + this.mediaGallery).select(record);
```

Reply:
Also can you please add `mediaGallery` property to defaults